themes/bootstrap3/templates/Auth/AbstractBase/newpassword.phtml

```diff
@@ -1,18 +1,14 @@
 <? if (isset($this->username)): ?>
   <div class="form-group">
-    <label class="col-sm-3 control-label"><?=$this->transEsc('Username') ?>:</label>
```

Comment:
The new password dialog looks weird and off-center to me now, especially with CAPTCHA on. I think we either need to lengthen the inputs or left-align the CAPTCHA. I'm seeing similar problems on related screens like create account and recover password.

Example:
![image](https://cloud.githubusercontent.com/assets/309069/26499280/01bd9210-4200-11e7-803d-96e8967d95b8.png)